SmileWeather/Classes/SmileWeatherDemoVC.h

```diff
@@ -23,6 +23,10 @@
 @property (nonatomic, getter= isFahrenheit) BOOL fahrenheit;
 /*!Night mode will change background color to black color.*/
 @property (nonatomic) BOOL nightMode;
+@property (nonatomic, nonnull) UIColor *mainInterfaceColor;
+@property (nonatomic, nonnull) UIColor *mainInterfaceColorNightMode;
+@property (nonatomic, nonnull) UIColor *higlightedInterfaceColor;
```

Comment:
@ckaas Thanks for your contribution! 

Just one place hope you can fix it.

Could you change `UIColor *mainInterfaceColorNightMode` to `UIColor *mainInterfaceNightModeColor`, I think it is more reasonable.